File: REST_DotNET_Coffee_Android/DTO/OrderDTO/OrderItemResponseDTO.cs
#nullable disable

public class OrderItemResponseDTO
{
    public String ProductName { get; set; }

    public List<String> IngredientList { get; set; }

    public int Quantity { get; set; }

    public double Price { get; set; }

    public double TotalPrice
    {
        get
        {
            return Price * Quantity;
        }
    }
}

File: Mobile_Coffee/Project-Mobile__Coffe-Store/app/src/main/java/com/nlu/packages/dotnet_callapi/responsedto/OrderItemResponseDTO.java
package com.nlu.packages.dotnet_callapi.responsedto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class OrderItemResponseDTO {
    private String productName;
    private List<String> ingredientList;
    private int quantity;
    private double price;
    public double getTotalPrice() {
        return this.price * this.quantity;
    }
}
